lib/membrane_rtc_engine/display_manager/display_manager.ex

```diff
@@ -115,7 +115,11 @@ defmodule Membrane.RTC.Engine.DisplayManager do
         &EndpointManager.add_tracks(&1, tracks, :outbound_tracks)
       )
 
-    {:noreply, %{state | endpoint_managers: endpoints}}
+    state = %{state | endpoint_managers: endpoints}
+
+    state = maybe_calculate_priority(state, [{:subscribe_on_tracks, endpoint_name} | state.vads])
+
+    {:noreply, state}
```

Comment:
I would avoid adding `subscribe_on_tracks` to trigger priority calculation. Instead we can add a flag to the `maybe_calculate_priority` like `force` that will indicate whether we should calculate priority despite the fact that there are no new vads.